lcls_tools/superconducting/scLinac.py

```diff
@@ -603,143 +607,156 @@ def setup_rf(self, desAmp):
                                     " range or use the rack large frequency scan"
                                     " to find the detune.")
-
+        self.set_chirp_range(200000)
+    
     def reset_interlocks(self, retry=True, wait=True):
         print(f"Resetting interlocks for {self} and waiting 3s")
         self.interlockResetPV.put(1, wait=True)
         if wait:
             sleep(3)
-
+        
         if retry:
             count = 0
             wait = 5
-            while caget(self.rf_permit_pv) == 0 and count < 3 and self.quench_latch_pv.value != 0:
+            while caget(self.rf_permit_pv) == 0 and count < 3:
                 print(f"{self} reset unsuccessful, retrying and waiting {wait} seconds")
                 self.interlockResetPV.put(1, wait=True)
                 sleep(wait)
                 count += 1
                 wait += 2
             if caget(self.rf_permit_pv) == 0:
                 raise utils.CavityFaultError(f"{self} still faulted after 3 reset attempts")
-
-    def runCalibration(self):
+    
+    def characterize(self):
         """
         Calibrates the cavity's RF probe so that the amplitude readback will be
         accurate. Also measures the loaded Q (quality factor) of the cavity power
         coupler
         :return:
         """
-
+        
         self.reset_interlocks()
-
+        
         print(f"setting {self} drive to {utils.SAFE_PULSED_DRIVE_LEVEL}")
         self.drivelevelPV.put(utils.SAFE_PULSED_DRIVE_LEVEL)
-
-        print(f"running {self} calibration")
-        utils.runCalibration(startPV=self.cavityCalibrationStartPV,
-                             statusPV=self.cavityCalibrationStatusPV,
-                             exception=utils.CavityQLoadedCalibrationError)
-
-        print(f"pushing {self} calibration results")
-        utils.pushAndSaveCalibrationChange(measuredPV=self.measuredQLoadedPV,
-                                           lowerLimit=self.loaded_q_lower_limit,
-                                           upperLimit=self.loaded_q_upper_limit,
-                                           pushPV=self.pushQLoadedPV,
-                                           savePV=self.saveQLoadedPV,
-                                           exception=utils.CavityQLoadedCalibrationError)
-
-        utils.pushAndSaveCalibrationChange(measuredPV=self.measuredCavityScalePV,
-                                           lowerLimit=utils.CAVITY_SCALE_LOWER_LIMIT,
-                                           upperLimit=utils.CAVITY_SCALE_UPPER_LIMIT,
-                                           pushPV=self.pushCavityScalePV,
-                                           savePV=self.saveCavityScalePV,
-                                           exception=utils.CavityScaleFactorCalibrationError)
-
-        print(f"{self} calibration successful")
-
+        
+        print(f"running {self} cavity characterization")
+        self.cavityCalibrationStartPV.put(1)
```

Comment:
to be consistent, might be good to rename this one to 'cavityCharacterizationStartPV'